eth_interface.md

```diff
@@ -6,7 +6,7 @@ The Ethereum Environment Interface exposes the core Ethereum API to the eWASM en
 
 We define the following Ethereum data types:
 - `bytes`: an array of bytes with unrestricted length
-- `address`: a 160 bit number, represented as a 20 bytes long little endian unsigned integer in memory
+- `address`: an array of 20 bytes
```

Comment:
Practically this means it is "big endian"? In evmc it still says big endian.

Reply:
I believe it means big endian, but I usually have trouble reasoning about 160-bit numbers.

Reply:
I don't think there is currently a case where a 160-bit type should be treated as a value.
The Solidity documentation states that only comparison operators are defined for the "address" type. This puts the type in a difficult situation where it is mostly not a quantity, but can be compared like an integer using operators like `<`, `>`, `<=`, and `>=`.
I think that the type should not have any notion of byte ordering other than defined by the hash function used to generate it. A correct address value should be the last 20 bytes of the hash of the public key, as usual.

Perhaps this reflects on a needed change in EVMC?

Reply:
Solidity actually considers them numbers (as they derive off `IntegerType`).

EVM considers them numbers too, since they are "right aligned" in the stack slot.